src/components/organisms/AdminVenueView/components/Spaces/Spaces.scss

```diff
@@ -9,7 +9,10 @@ $icon-size: 20px;
   flex: 1;
 
   &__rooms {
+    @include scrollbar;
     width: 20%;
+    height: 88vh;
```

Comment:
You can't use specific value, the scroll will not scroll till the end on many resolutions.

Reply:
You need to put all of the content except for the footer in another `div` and add styles to it.

Reply:
fixed in 4b5b230d9f486566455287d7401112187b5e550f

Reply:
The fix provided in https://github.com/sparkletown/sparkle/commit/4b5b230d9f486566455287d7401112187b5e550f is using media queries which will not work correctly for all of the resolutions. I changed my laptop resolution to 1336x768 and some of the scroll at the bottom is cut off. 

You need to put all of the content in a single div and give it 100% height with overflow scroll, that way it will scroll correctly and look the same on every screen.

Reply:
fixed in a8cf51fb62916ec11f3784e1b09328ca3e42e604